test/integration/model/create.test.js

```diff
@@ -1852,6 +1852,33 @@ describe(Support.getTestDialectTeaser('Model'), function() {
         expect(m.secret).to.be.eql(M2.secret);
       });
     });
+    it('should return autoIncrement primary key', function() {
+      var Maya = this.sequelize.define('Maya', {
+        name: Sequelize.STRING,
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true
+        },
+        createdAt: {
+            field: 'created_at',
+            type: Sequelize.DATE
+        },
+        updatedAt: {
+            field: 'updated_at',
+            type: Sequelize.DATE
+        }
```

Comment:
Are all those column definitions needed to reproduce the bug? Or can we omit createdAt/updatedAt and maybe even id?